tests/pageobjects/PreJoinScreen.ts

```diff
@@ -0,0 +1,31 @@
+import PreMeetingScreen from './PreMeetingScreen';
+
+const DISPLAY_NAME_ID = 'premeeting-name-input';
+const JOIN_BUTTON_TEST_ID = 'prejoin.joinMeeting';
+
+/**
+ * Page object for the PreJoin screen.
+ */
+export default class PreJoinScreen extends PreMeetingScreen {
+    /**
+     * Returns the join button element.
+     */
+    getJoinButton(): ChainablePromiseElement {
+        return this.participant.driver.$(`[data-testid="${JOIN_BUTTON_TEST_ID}"]`);
+    }
+
+    /**
+     * Returns the display name input element.
+     */
+    getDisplayNameInput(): ChainablePromiseElement {
+        return this.participant.driver.$(`#${DISPLAY_NAME_ID}`);
+    }
+
+    /**
+     * Waits for pre join screen to load.
+     */
+    async waitForLoading() {
```

Comment:
Is this method used somewhere? I don't seem to find any references.

Reply:
https://github.com/jitsi/jitsi-meet/blob/7f0a1f845aa542ef392ecdc661ab18915b058c3a/tests/specs/3way/lobby.spec.ts#L435